src/current/v24.3/create-logical-replication-stream.md

```diff
@@ -0,0 +1,154 @@
+
+## Bidirectional LDR
+
+_Bidirectional_ LDR consists of two clusters with two LDR jobs running in opposite directions between the clusters. If you're setting up bidirectional LDR, both clusters will act as a source and a destination in the respective LDR jobs. 
+
+LDR supports starting with two empty tables, or one non-empty table. LDR does **not** support starting with two non-empty tables. When you set up bidirectional LDR, if you're starting with one non-empty table, start the first LDR job from empty to non-empty table. Therefore, you would run `CREATE LOGICAL REPLICATION STREAM` from the destination cluster where the non-empty table exists.
+
+## Examples
+
+To start LDR, you must run the `CREATE LOGICAL REPLICATION STREAM` statement from the **destination** cluster. Use the[ fully qualified table name(s)]({% link {{ page.version.version }}/sql-name-resolution.md %}#how-name-resolution-works) and ensure that the source and destination table names are identical. The following examples show statement usage with different options and use cases.
+
+### Start an LDR stream
+
+{% include {{ page.version.version }}/ldr/multiple-tables.md %}
+
+#### Single table
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+~~~
+
+#### Multiple tables
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLES ({database.public.table_name},{database.public.table_name},...)  ON 'external://{source_external_connection}' INTO TABLES ({database.public.table_name},{database.public.table_name},...);
+~~~
+
+### Ignore row-level TTL deletes
+
+If you would like to ignore [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) deletes in the LDR stream, set the [`ttl_disable_changefeed_replication` storage parameter]({% link {{ page.version.version }}/row-level-ttl.md %}#ttl-storage-parameters) on the table. On the **source** cluster, alter the table to set the table storage parameter:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER TABLE {table_name} SET (ttl_disable_changefeed_replication = 'true');
+~~~
+
+When you start LDR on the **destination cluster**, include the `discard = ttl-deletes` option in the statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name} WITH discard = ttl-deletes;
+~~~
+
+### Bootstrap LDR from a backup
+
+To start LDR from a specific point in time without an initial backfill of the destination table, you can use a backup and the `cursor` option with `CREATE LOGICAL REPLICATION STREAM`.
```

Comment:
given some recent developments in this approach, i think we should drop this section for now, right @alicia-l2 ?

Reply:
Removed for now, after discussing with @alicia-l2